docs/ancestry.rst

```diff
@@ -957,18 +957,22 @@ processes. In particular, simulations run concurrently in subprocesses using
 seeds by default.
```

Comment:
Something strange has happened here - did you pick up some extra stuff in a rebase? This PR probably shouldn't touch this file.

Reply:
This section of the docs gave different HTML every time the docs were built. I know it isn't ideal, but this was the only way I could see to get the point across about randomness while using deterministic HTML.

Reply:
I see, sounds good.